OnionAppWithCQRS/Domain/OfficeAddEntity.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Domain
+{
+    public class OfficeAddEntity
```

Comment:
Strange entity. 
Is it really Entity?
Where is Id?

Reply:
rename